tests/consumers/test_extract_data_rmq.py

```
@@ -123,12 +123,12 @@ async def __aexit__(inner, exc_type, exc, tb):
                     return False 
             return Ctx()
     
-    req_payload = {"userId": "user123", "s3Key": "path/to/file.html"}
+    req_payload = {"userId": 5, "s3Key": "path/to/file.html"}
     body_bytes = json.dumps(req_payload).encode()
     msg = DummyMessage(body_bytes)
     
     with pytest.raises(Exception):
```

Ruff (0.8.2)

`pytest.raises(Exception)` should be considered evil

(B017)